edge-agent/docker/linux/arm32v7/base/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt-get update && \
 
 # create a user to allow agent to optionally run as non-root
 ARG EDGEAGENTUSER_ID
-ENV EDGEAGENTUSER_ID ${EDGEAGENTUSER_ID:-1000}
+ENV EDGEAGENTUSER_ID ${EDGEAGENTUSER_ID:-13622}
```

Comment:
Assuming you created a new base image for master, the Dockerfile one directory up should be updated with that new image tag.

Reply:
Done!